jazzy/AzureCommunicationUICalling.yml

```diff
@@ -3,7 +3,7 @@ author_url: https://azure.github.io/azure-sdk/
 github_url: https://github.com/Azure/communication-ui-library-ios
 module: AzureCommunicationUICalling
 module_version: 1.1.0
-readme: ../build/communication-ui-library-ios/README.md
+readme: ../build/communication-ui-library-ios/AzureCommunicationUI/sdk/AzureCommunicationUICalling/README.md
```

Comment:
these links to readme might not be available until we merge chat to main on release day

Reply:
That's fine as long as we don't try to regenerate the docs until then. So as long as UICalling doesn't release beforehand there won't be an issue.